src/shared/sam/activation.ts

```diff
@@ -65,11 +65,11 @@ export async function activate(ctx: ExtContext): Promise<void> {
         )
     )
 
-    await detectSamCli({ passive: true, showMessage: false })
```

Comment:
Do not do `detectSamCli` at startup. Because:

1. we don't want to force-update the user setting
2. if user intentionally has set SAM CLI to a "invalid" path, we don't want to show a warning message every time